test/base-url-env-var.js

```diff
@@ -0,0 +1,36 @@
+const t = require('../test-lib/test.js');
+const assert = require('assert');
+let apos;
+let savedBaseUrl;
+
+const config = {
+  root: module,
+  // Should get overridden by the above
+  baseUrl: 'http://localhost:3000'
+};
+
+describe('Locales', function() {
+  this.timeout(t.timeout);
+
+  before(async function() {
+    savedBaseUrl = process.env.APOS_BASE_URL;
+    process.env.APOS_BASE_URL = 'https://madethisup.com';
+    apos = await t.create(config);
+  });
+
+  after(function() {
+    if (savedBaseUrl) {
```

Comment:
`savedBaseUrl` is useless here, you can use `process.env.APOS_BASE_URL` directly